README.md

```diff
@@ -69,31 +69,30 @@ _One common template for creating a user story is the Who, What, Why template: A
-| selectedSongId   | String | unique song id from Spotify                      |
-| mood             | String | user inputted mood                               |
-| review           | String | user inputted review for the song                |
-| rating           | String | user inputted rating for the song (1-5)          |
-| userId           | String | users unique Spotify username                    |
-| selectedSongUrl  | String | song image URL from api search call              |
-| selectedSongName | String | song title from api search                       |
-| likes            | Number | number of likes for the post                     |
-| comments         | Array  | array of comment objectIds that have been posted |
+| Property       | Type   | Description                                      |
+| -------------- | ------ | ------------------------------------------------ |
+| objectId       | String | unique id for the user post (default field)      |
+| createdAt      | Date   | (default auto generated field)                   |
+| updatedAt      | Date   | (default auto generated field)                   |
+| selectedSongId | String | unique song id from Spotify                      |
+| mood           | String | user inputted mood                               |
+| review         | String | user inputted review for the song                |
+| rating         | String | user inputted rating for the song (1-5)          |
+| userId         | String | users unique Spotify username                    |
+| likes          | Array  | array of Like objectIds that have been posted    |
+| comments       | Array  | array of Comment objectIds that have been posted |
 
 #### Comments
 
-| Property       | Type    | Description                                 |
-| -------------- | ------- | ------------------------------------------- |
-| objectId       | String  | unique id for the user post (default field) |
-| createdAt      | Date    | (default auto generated field)              |
-| updatedAt      | Date    | (default auto generated field)              |
-| comment        | String  | user inputted comment for each post         |
-| selectedSongId | String  | unique song id from Spotify                 |
-| postId         | Pointer | pointer to unique id for the user post      |
+| Property       | Type    | Description                                  |
+| -------------- | ------- | -------------------------------------------- |
+| objectId       | String  | unique id for the user post (default field)  |
+| createdAt      | Date    | (default auto generated field)               |
+| updatedAt      | Date    | (default auto generated field)               |
+| comment        | String  | user inputted comment for each post          |
+| selectedSongId | String  | unique song id from Spotify                  |
+| postId         | Pointer | pointer to unique id for the user post       |
```

Comment:
What is this postId? Is this the objectId in the post table?

Reply:
Yes this postId is the objectId in the post table. Would a name like postObjectId maybe be better?

Reply:
You can rename the column to postId. That way the object/entity this column is referring to would be more clear.